wordpress-mobile/WordPress-iOS
# Rich Notifications : Context Extension scaffold

Fixes #9844, which calls for the creation & configuration of a notification content extension.

To test:

- Verify that the branch builds & existing tests pass.
- While debugging the content extension, manually push a payload with a valid `note_id` value. Instructions for this can be found here: #9924 
- Force-touch on the _Short Look_ to present the _Long Look_. You should see the original notification body, along with a green bar : that's a remnant from the default storyboard content.

![9844](https://user-images.githubusercontent.com/221062/44238164-5a646600-a168-11e8-9624-db0d5c8307bf.png)
